temporalcli/commandsgen/commands.yml

```diff
@@ -3284,6 +3253,7 @@ option-sets:
           Assume inputs are base64-encoded and attempt to decode them.
 
   - name: update-starting
+    description: Workflow update commands
```

Comment:
Same here. "Commands that update Workflows" for alignment

Reply:
Also, looking down to 3288, is this correct? Is this "Commands that use updates to start Workflows"?

Reply:
removing `description` [per comment above](https://github.com/temporalio/cli/pull/693/files#r1803621328)